docs/reference/cli/bc_notify_subscribe.md

```
@@ -0,0 +1,47 @@
+bc notify subscribe telegram:bc-dev eng-02                   # Telegram group
+```
+
+### Description
+
+Creates a subscription linking an agent to a notification source. Once subscribed, the agent receives platform events dispatched by the notification service.
+
+Use `--mention-only` to filter notifications so the agent only receives events where it is explicitly @mentioned. This is useful for noisy channels where the agent should only respond when addressed directly.
+
+The source must be a valid `platform:channel` identifier discovered by a connected gateway adapter. Use `bc notify list` to see available sources.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)